ch02/README.md

```diff
@@ -437,7 +437,7 @@ i = ic;     // legal.
 p1 = p3;    // illegal. p3 is a pointer to const int.
 p1 = &ic;   // illegal. ic is a const int.
 p3 = &ic;   // legal.
-p2 = p1;    // legal.
```

Comment:
yes, you are right.

If we assume `p1`, `p2`, `p3` all have been initialized. `p3 = &ic` is also an error, cause `p3` is `const int *const` type. right?